### ISSUE TITLE
[feature-request] Add node-postgres support

### ISSUE THREAD

**Issue by pawsaw** [Microsoft/TypeScript-Node-Starter#48]
It would be great to have support for some more traditional DBMS ...  maybe on a separate branch for each DBMS?

In this ticket I'd like to have postgres supported, using this great npm module:

https://node-postgres.com/

Typings dbd.

 Great work by the way.

**Comment by devnetics** [comment]
I don't think that adding multiple DBMS systems in various branches is a very maintainable strategy. You would probably need to maintain that on your own, or fork it.

**Comment by tioback** [comment]
How about a new branch where [Loopback](https://loopback.io/) is used, so you could use whatever DBMS you felt like using?

**Comment by devnetics** [comment]
Would probably be a bit more OSS friendly to add something like knex.js/bookshelf.

**Comment by devnetics** [comment]
(not the LB isn't, I just don't think you want to tie users to a platform for a starter repo).

**Comment by welkie** [comment]
I like the fact that this is just a starter repo, agnostic with respect to what people want to create from it. Myself, I've used it to start a few projects. Some used PostgreSQL with TypeORM and one used MongoDB (without Mongoose). But I wouldn't want that added in, because that wouldn't be fair to other people who might want to go in a different direction.

The starter repo being agnostic helps people see a clearer path to what they want to create instead of having to think about what surgery to perform on it to remove what they don't want. (at least that's my opinion, I'd love to hear others too)
